docs/source/glossary.rst

```diff
@@ -4,9 +4,7 @@ Glossary
 .. glossary::
 
     ADS
-    MTD
-        Analysis Data Service (ADS, aka Mantid Data (MTD)), the internal database held by :term:`Mantid` of all *named* workspaces.
-        Any workspace referenced "by name", usually by a Mantid algorithm, needs to be registered in this database.
```

Comment:
This definition needs to remain

Reply:
This definition is retained. Just moved it elsewhere because it wasn't alphabetical.